ceph-pull-requests-arm64/config/definitions/ceph-pull-requests-arm64.yml

```diff
@@ -9,7 +9,7 @@
     concurrent: true
     disabled: false
     name: ceph-pull-requests-arm64
-    node: 'arm64 && !centos7 && !centos8 && !centos9'
+    node: 'arm64 && (jammy || focal) && !centos7 && !centos8 && !centos9'
```

Comment:
presumably, we have some nodes stuck with ubuntu bionic, i think the right fix is to upgrade these ancient installations to the latest LTS, i.e., jammy.

Reply:
yes, I agree, better to do that.